### PR TITLE
Dashboard: Improve import page easy of use

### LINKED ISSUE #74319: Improve import component adding better links to community templates
**Problem**
The current import view can be confusing for first-time users in different aspects:
* There is an input, inviting you to download/find dashboards from grafana.com but there is no link there
* There is a big input box referring to panel JSON when in reality is dashboard JSON

Captura de pantalla 2023-09-05 a las 14 51 29

**Solution**
* Improve the copies
* Add links to dashboard collection
* Add placeholder to the textarea to help the user understand what dashboard JSON model is

### PR DESCRIPTION
**Pull request by ivanortegaalba**
|Before|After
|-|-|
Captura de pantalla 2023-09-05 a las 15 06 12|Captura de pantalla 2023-09-07 a las 16 14 42

Fixes #74319

### CODE REVIEW

**Comment by ivanortegaalba** [comment]
@imatwawana @Ijin08 I just updated the UI with the following

Captura de pantalla 2023-09-05 a las 14 56 27

Any thoughts? Anything else we can improve?

I see:
* Inconsistency between the labeling of the different input elements
* Not very descriptive page description

WDYT? Please give me any proposals to make this UI better :)

File: public/app/features/manage-dashboards/DashboardImportPage.tsx [hunk 1]
```diff
@@ -136,7 +144,15 @@ class UnthemedDashboardImport extends PureComponent<Props> {
           <Form onSubmit={this.getGcomDashboard} defaultValues={{ gcomDashboard: '' }}>
             {({ register, errors }) => (
               <Field
-                label="Import via grafana.com"
+                label={
+                  <p>
+                    Find and import a dashboard{' '}
+                    <TextLink href="https://grafana.com/grafana/dashboards/" external>
+                      for common applications at grafana.com/dashboards
+                    </TextLink>
```
Comment (imatwawana) [inline public/app/features/manage-dashboards/DashboardImportPage.tsx:152]:
```suggestion
                    Find and import dashboards for common applications at{' '} 
                    <TextLink href="https://grafana.com/grafana/dashboards/" external>grafana.com/dashboards
                    </TextLink>
```

**Review by imatwawana — COMMENTED** [review]
I've suggested different placement for the link text. 

Also confirming I didn't find any docs impact (text or screenshots) in making this change.

**Review by imatwawana — APPROVED** [review]
LGTM!

**Review by polibb — APPROVED** [review]
thank you!!

**Pull request merged**